kubernetes/INSTRUCTION.md

```diff
@@ -0,0 +1,218 @@
+## Torch elastic k8s controller
```

Comment:
This should be the README. And please add the source file structure here since that's really useful as well.

Reply:
Yeah, it is. In that folder, we already have one README.md and that's the reason I have a different name. Let me file a separate PR to move native examples to `examples/` and this will be clean for operator codes

Reply:
Once https://github.com/pytorch/elastic/pull/68 is merged, I will rebase the change and rename this file to README.md